metalus-core/src/main/scala/com/acxiom/pipeline/PipelineStepMapper.scala

```diff
@@ -278,7 +285,10 @@ trait PipelineStepMapper {
   }
 
   private def getPipelineParameterValue(pipelinePath: PipelinePath, pipelineContext: PipelineContext): Option[Any] = {
-    val paramName = pipelinePath.mainValue.substring(1)
+    val paramName = pipelinePath.mainValue.toLowerCase match {
+      case "@laststepid" | "#laststepid" => pipelineContext.getGlobalString("lastStepId").getOrElse("")
```

Comment:
Create a test to verify this will not break when encountering an OR statement:

!someValueNotPresent || @LastStepId